src/Compilers/CSharp/Portable/Lowering/LambdaRewriter/LambdaRewriter.cs

```diff
@@ -738,6 +795,129 @@ public override BoundNode VisitBaseReference(BoundBaseReference node)
                 : FramePointer(node.Syntax, _topLevelMethod.ContainingType); // technically, not the correct static type
         }
 
+        /// <summary>
+        /// Rewrites a reference to an unlowered local function to the newly
+        /// lowered local function.
+        /// </summary>
+        private void RemapLocalFunction(
+            SyntaxNode syntax,
+            MethodSymbol localFunc,
+            out BoundExpression receiver,
+            out MethodSymbol method,
+            ref ImmutableArray<BoundExpression> parameters)
```

Comment:
This wasn't changed in this PR, but should this be `arguments`? (same with `parametersBuilder`, comments, etc.)